From 40dbce754c20fe9a303f295d67d65d1611cd3b94 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:35 -0400
Subject: [redhat] kabi: Adding symbol bio_put (fs/bio.c)

Message-id: <21560260548a829161b1609209c5c44d12adbeac.1410510773.git.poros@redhat.com>
Patchwork-id: 95915
O-Subject: [PATCH RHEL7.1 035/233] kabi: Adding symbol bio_put (fs/bio.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Release a reference to a bio.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bio_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bio_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bio_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bio_put  b/redhat/kabi/kabi-module/kabi_ppc64/bio_put 
new file mode 100644
index 0000000..75e0505
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bio_put  
@@ -0,0 +1,2 @@
+#1-
+0x7dec7b53 bio_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bio_put  b/redhat/kabi/kabi-module/kabi_s390x/bio_put 
new file mode 100644
index 0000000..8e85cd5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bio_put  
@@ -0,0 +1,2 @@
+#1-
+0x846c3efe bio_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bio_put  b/redhat/kabi/kabi-module/kabi_x86_64/bio_put 
new file mode 100644
index 0000000..6feb4fa
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bio_put  
@@ -0,0 +1,2 @@
+#1-
+0x867c68af bio_put vmlinux EXPORT_SYMBOL
-- 
1.7.1